### ISSUE TITLE
Wii BRLYT Textures didn't loading

### ISSUE THREAD

**Issue by ZAZiOs** [KillzXGaming/Switch-Toolbox#512]
While i was translating Rhythm Heaven Fever to Russian i had an issue. 
Textures **loaded once,** and didnt do it after.
Animation works fine. I had all TPL's that programm asks for, but i cant add them. I tried drag & drop, tried adding it by pressing + button. Even add button on materials texture viewer didn't work either.
![image](https://user-images.githubusercontent.com/85897688/189889972-a734141e-0461-4702-adc6-7c6e8d80db6d.png)
![image](https://user-images.githubusercontent.com/85897688/189890000-778de41f-760f-452d-9efa-4934e74cab8e.png)

**Comment by KillzXGaming** [comment]
Might be that it has 2 .tpl extensions? Not sure if that is normal or not.

**Comment by ZAZiOs** [comment]
> Might be that it has 2 .tpl extensions?

No, its not because of that... Well i'll be waiting for the gc textures feature ^^
